neondatabase/website
# About us page update

This pull request brings design update for About us page.

Steps to check:

- open [preview](https://neon-next-git-about-us-page-upd-neondatabase.vercel.app/about-us)
- make sure that everything looks and works as expected

<details>
<summary>Design</summary>

<img width="1915" alt="about-us-1920-adaptive" src="https://github.com/user-attachments/assets/b74b2039-f02b-440a-afd3-e908055ece2c">
</details>
